test/unit/ngsiv2/examples/contextRequests/updateContextMultientityPlugin15.json

```diff
@@ -0,0 +1,25 @@
+{
+  "actionType": "update",
+  "entities": [
+    {
+      "id": "gps1",
+      "type": "GPS"
+    },
+    {
+      "explicit": {
+        "type": "number",
+        "value": 52
+      },
+      "type": "GPS",
+      "id": "SO5"
+    },
+    {
+      "explicit": {
+        "type": "number",
+        "value": 13
+      },
+      "type": "GPS",
+      "id": "SO6"
+    }
+  ]
+}
```

Comment:
This is a particular case in which the main entity does not have any attribute. I raised an Issue in order to discuss if this makes sense or not (https://github.com/telefonicaid/iotagent-node-lib/issues/1167)

NTC